client/index.html

```diff
@@ -478,7 +478,7 @@ <h1 class="title">Help</h1>
 								</p>
 								<p>
 									A color reference can be found
-									<a href="http://www.mirc.com/colors.html" target="_blank">here</a>.
+									<a href="http://modern.ircdocs.horse/formatting.html#colors" target="_blank" rel="noopener">here</a>.
```

Comment:
Ah, didn't see the ping here, need to reduce the amount of stuff I'm following :)

Definitely, no problem! I'll yell in that other issue once it's sorted.